### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by danielchen2003**
Put api we used into Readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
 # Solar Conveyance
 Our goal is to give you news about the events happening in our solar system. We'll tell you about the weather on mars and other weather phenomena in space. We'll also let you know about current space shuttle launches. We hope to give you a better appreciation for being apart of this beautiful universe of ours.
 
+
+##APIs We Used
```
Comment (dominik-ilja) [inline README.md]: Could you make this a `#` instead of `##`

**Review by dominik-ilja — COMMENTED** [review]
Could you change the heading from `#` to `##`

**Comment by danielchen2003** [comment]
Changed to singe # already

**Review by dominik-ilja — APPROVED** [review]
Awesome! Thank you!

**Pull request merged**